tests/test_routes.py

```diff
@@ -141,7 +141,34 @@ def test_read_order(self):
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
         data = resp.get_json()
         self.assertEqual(data["customer_name"], order.customer_name)
+        
+    
+    def test_update_order(self):
```

Comment:
This test case failed during "make test", maybe there are some issues that need to be revised.

Reply:
Fixed now! I was facing docker issues so it wasn't working as expected.